workflow/controller/operator.go

```diff
@@ -913,6 +920,10 @@ func (woc *wfOperationCtx) createPVCs() error {
 	return nil
 }
 
+func isPVCExistErr(err error) bool {
+	return strings.Contains(err.Error(), "persistentvolumeclaims") && strings.Contains(err.Error(), "already exists")
```

Comment:
There is an API library provided by kubernetes to detect the already exists error that is not as fragile as this. Also this change does not implement the requirement of checking ownership reference before ignoring the error.